modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/Activities/index.scss

```diff
@@ -0,0 +1,111 @@
+.page-activities {
```

Comment:
Sort properties if possible,

for example

`.border-header` should come before `.font-table`

Reply:
Good. I didn't know.
Thanks Mestre Kevin!